armi/tests/Godiva-blueprints.yaml

```diff
@@ -0,0 +1,733 @@
+        shape: RadialSegment
+        material: Void
+        Tinput: 0.0
+        Thot: 0.0
+        outer_theta: 0.7853981633974483
+        height: 3.5000000000000004
+        inner_theta: 0.0
+        outer_radius: 6.002
+        inner_radius: 3.001
+        mult: 0.4045467972755432
+    - name: block2_1_2
+      Godiva:
+        shape: RadialSegment
+        material: UZr
+        Tinput: 26.85
+        Thot: 26.85
+        outer_theta: 0.7853981633974483
+        height: 3.5000000000000004
+        inner_theta: 0.0
+        outer_radius: 6.002
+        inner_radius: 3.001
+        mult: 1.0
+    - name: block2_1_3
+      Godiva:
+        shape: RadialSegment
+        material: UZr
+        Tinput: 26.85
+        Thot: 26.85
+        outer_theta: 0.7853981633974483
+        height: 3.5000000000000004
+        inner_theta: 0.0
+        outer_radius: 6.002
+        inner_radius: 3.001
+        mult: 1.0
+    - name: block2_1_4
+      Godiva:
+        shape: RadialSegment
+        material: UZr
+        Tinput: 26.85
+        Thot: 26.85
+        outer_theta: 0.7853981633974483
+        height: 3.5000000000000004
+        inner_theta: 0.0
+        outer_radius: 6.002
+        inner_radius: 3.001
+        mult: 1.0
+    - name: block2_1_5
+      Godiva:
+        shape: RadialSegment
+        material: UZr
+        Tinput: 26.85
+        Thot: 26.85
+        outer_theta: 0.7853981633974483
+        height: 3.5000000000000004
+        inner_theta: 0.0
+        outer_radius: 6.002
+        inner_radius: 3.001
+        mult: 0.5924441814422607
+      compliment:
+        shape: RadialSegment
+        material: Void
+        Tinput: 0.0
+        Thot: 0.0
+        outer_theta: 0.7853981633974483
+        height: 3.5000000000000004
+        inner_theta: 0.0
+        outer_radius: 6.002
+        inner_radius: 3.001
+        mult: 0.40755581855773926
+    height:
+    - 3.5000000000000004
+    - 3.5000000000000004
+    - 3.5000000000000004
+    - 3.5000000000000004
+    - 3.5000000000000004
+    axial mesh points:
+    - 5
+    - 5
+    - 5
+    - 5
+    - 5
+    radial mesh points: 2
+    azimuthal mesh points: 7
+    material modifications:
+      U235_wt_frac:
+      - 0.9371
+      - 0.9371
+      - 0.9371
+      - 0.9371
+      - 0.9371
+      ZR_wt_frac:
+      - 0.0
+      - 0.0
+      - 0.0
+      - 0.0
+      - 0.0
+    xs types:
+    - A
+    - A
+    - A
+    - A
+    - A
+  assembly3_1:
+    specifier: assembly3_1
+    blocks:
+    - name: block3_1_1
+      Godiva:
+        shape: RadialSegment
+        material: UZr
+        Tinput: 26.85
+        Thot: 26.85
+        outer_theta: 0.7853981633974483
+        height: 3.5000000000000004
+        inner_theta: 0.0
+        outer_radius: 9.0
+        inner_radius: 6.002
+        mult: 0.046154800802469254
+      compliment:
+        shape: RadialSegment
+        material: Void
+        Tinput: 0.0
+        Thot: 0.0
+        outer_theta: 0.7853981633974483
+        height: 3.5000000000000004
```

Comment:
This screams "Python rounding error" to me.

Reply:
I moved to 3.5